src/mixins/Projector.ts

```diff
@@ -286,6 +289,24 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(Base: T)
 			return this._projection.domNode.outerHTML;
 		}
 
+		protected afterRootCreateCallback(element: HTMLElement,
```

Comment:
I think this formatting is really funky :) is it different from the formatting in widget base?

Reply:
I think you're really funky

Reply:
Formatting is the same as in widgetbase

Reply:
it's not the same, it is like this in widget base

```ts
	protected afterRootCreateCallback(
		element: HTMLElement,
  		projectionOptions: ProjectionOptions,
  		vnodeSelector: string,
  		properties: VNodeProperties
  	): void {
              // ... stuff
  	}
```

Reply:
notice the first param is on the next line?

Reply:
my apologies, have fixed